intl/en/messages.json

```diff
@@ -43,6 +46,7 @@
   "514985b2327f061ffb1c932f6b909979": "Model {0} is not defined.",
   "cc09e361de7b0f3dc1687515c5713885": "Property names containing a dot are not supported. Model: {0}, property: {1}",
   "da02dd6c53d4148320eeb31718a7aebe": "Invalid type for property {0}",
+  "ddf0aa14803f1c84f4a97f3803f7471c": "Class name required",
```

Comment:
AFAICT, this line was moved from L34 to L49. This looks like a short-coming of strong-globalize, see https://github.com/strongloop/strong-globalize/issues/74